js/main.js

```diff
@@ -322,6 +322,29 @@
 			}
 		});
 
+		$(document).keydown(function (event) {
```

Comment:
can you please add a little comment on top which does describe what this code part is about